tests/MongoDB.Bson.Tests/Jira/CSharp301Tests.cs

```diff
@@ -60,7 +60,10 @@ public void TestHashtableEmpty()
         {
             var c = new C { Id = 1, Obj = new Hashtable { } };
             var json = c.ToJson();
-#if NET452
+#if NET452 || NETCOREAPP3_0
+            // Hashtable is situated in well-known libraries for:
+            // - NET452: mscorlib
+            // - NETCOREAPP3_0: System.Private.CoreLib
             var discriminator = "System.Collections.Hashtable";
 #else
             var discriminator = typeof(Hashtable).AssemblyQualifiedName;
```

Comment:
This is an ugly problem, but unrelated to this PR. We shouldn't be using a default discriminator that changes from one framework version to another. But I suppose that ship already sailed long ago.

Reply:
agreed, I mentioned it here: https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/162#discussion_r529840987

Reply:
not sure whether we want to make any changes here (in the scope of a different ticket). Most-likely no